catroid/src/main/res/values/strings.xml

```diff
@@ -1052,6 +1052,13 @@
     <string name="category_assertions">Testing</string>
     <!--  -->
 
+    <!-- User Defined Bricks -->
+    <string name="brick_user_defined">Your brick</string>
+    <string name="brick_user_defined_add_label">Add label</string>
+    <string name="brick_user_defined_add_input">Add input</string>
+    <string name="brick_add_new_user_brick">Add new User Brick</string>
```

Comment:
```suggestion
    <string name="brick_add_new_user_brick">Add new brick</string>
```

*Reason:* The term "user brick" is only relevant for our developers, but shall not be visible for our users. Also, we try to use normal English capitalization rules.